library_generation/test/resources/proto_path_list.txt

```diff
@@ -17,7 +17,7 @@ google/cloud/redis/v1 java-redis false
 google/cloud/redis/v1beta1 java-redis false
 # google/example/library/v1 google-cloud-example-library-v1-java null false
 google/devtools/containeranalysis/v1 java-containeranalysis false
-google/iam/v1 java-iam false
+google/iam/v2 java-iam false
```

Comment:
Could you add another line for `google/iam/v2` because I use `google/iam/v1` to test proto-only library.

Reply:
I'll disable `v1` for the post-processing IT and add `v2` separately. The `iam` [owlbot yaml](https://github.com/googleapis/google-cloud-java/blob/18020600f38e9cf966d3b4c12035954a7124344a/java-iam/.OwlBot.yaml#L25-L28) only supports `v2` and `v2beta1` so using `v1` will fail to move any java files into the workspace